src/Toolkit/Toolkit.Maui/UtilityNetworkTraceTool/UtilityNetworkTraceTool.Appearance.cs

```diff
@@ -86,22 +106,22 @@ static UtilityNetworkTraceTool()
         <RowDefinition Height=""*"" />
     </Grid.RowDefinitions>
     <Frame x:Name=""{nameof(PART_NoNetworksWarning)}"" BorderColor=""{{AppThemeBinding Light=#D83020, Dark=#FE583E}}"" CornerRadius=""4"" Margin=""4"" Grid.RowSpan=""3"" IsVisible=""false"">
-        <Label Text=""No utility networks found."" />
+        <Label Text=""{noUtilityNetworks}"" />
```

Comment:
Actually, I am not sure what symbols will be used in the translation and sounds like a good idea to replace `""` with `&quot;`. I can't seem to find `cs` file with `&quot;` in toolkit repo but our targets file use it.

@dotMorten,  is this something we'll do for the other MAUI XAML code? Should I address this now in this PR or next round? 

I found a few .NET Maui toolkit implementation issues that I'd like to address in the future because it affected my dev summit demo and I ended up writing my own to match WPF implementation.

- previous traces are cleared
- no color palette to compare aggregated geometry results
- no terminal picker or fraction along slider
- theme issues

If this feedback can wait til I get around to fixing MAUI trace tool issues, I can tackle it then.